pkg/cmd/build/authprovider.go

```diff
@@ -94,3 +101,20 @@ func (ap *authProvider) Credentials(_ context.Context, req *auth.CredentialsRequ
 	}
 	return res, nil
 }
+
+func isErrCredentialsHelperNotAccessible(err error) bool {
```

Comment:
This is taking a dependency on the error message that might change. Can we add a unit test where we trigger the auth error to guard us against string changes?

Reply:
done [here](https://github.com/okteto/okteto/blob/jlopezbarb/omit_auth_error/pkg/cmd/build/authprovider_test.go#L10)

Reply:
added -> https://github.com/okteto/okteto/pull/3374/files#diff-1fb806e1dce09810a22590fe2d06868d1c0ac91f098506d101ebf73fb5f5d357R51